pages/index.js

```diff
@@ -185,7 +185,7 @@ export default function Home() {
 			</div>
 			</div>
 			<div id='sponsors' className='mt-20'>
-				<Sponsors imgs={['/img/apidays.png']} />
+				<Sponsors imgs={[{image:'/img/apidays.png',sponsorUrl:"https://www.apidays.global/"}]} />
```

Comment:
Can you extend this prop to include the remaining sponsors?

Reply:
Hi @AceTheCreator,

Should I create a separate prop for the financial sponsors instead of keeping them hardcoded as they are now, or anything else?

Reply:
Hi @AceTheCreator,

I've updated the component to extend the imgs prop, now including both financial and event sponsors. Let me know if there's anything specific you'd like me to adjust further!